lib/internal/http2/core.js

```diff
@@ -332,15 +424,15 @@ class Http2Session extends EventEmitter {
     // streams, these may all need to be made aware of changes
     // in state that occur -- such as when the associated socket
     // is closed. To do so, we need to set the max listener count
-    // to something more reasonable.
+    // to something more reasonable to avoid the memory leak warning.
```

Comment:
avoiding a warning is not newbie friendly, as it encourages a good practice. The actual reason is that we can have an extremely high number of streams multiplexed over the session.